resources/lang/zh/accounts.php

```diff
@@ -20,7 +20,7 @@
 
 return [
     'edit' => [
-        'title' => '账户设置',
+        'title' => '<strong>账户</strong>设置',
```

Comment:
it's just ok for chinese:
![image](https://user-images.githubusercontent.com/18349191/28248129-bf2f8e18-6a71-11e7-92e7-76c9513f92bd.png)

@Robwolver what do you think?